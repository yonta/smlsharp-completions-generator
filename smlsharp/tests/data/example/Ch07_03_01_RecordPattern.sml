fun f {X = x, Y = y} = (x, y);

fun f {X = x, Y = y, ...} = (x, y);

fun f {X, Y, ...} = (X,Y);

fun f ({X,...}::_) = X;

