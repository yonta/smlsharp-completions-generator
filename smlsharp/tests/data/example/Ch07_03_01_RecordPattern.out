val f = fn : ['a, 'b. {X: 'a, Y: 'b} -> 'a * 'b]

val f = fn : ['a#{X: 'b, Y: 'c}, 'b, 'c. 'a -> 'b * 'c]

val f = fn : ['a#{X: 'b, Y: 'c}, 'b, 'c. 'a -> 'b * 'c]

val f = fn : ['a#{X: 'b}, 'b. 'a list -> 'b]
